CMakeLists.txt

```diff
@@ -1651,10 +1616,10 @@ if(WIN32)
     hpx_add_target_compile_option(-Ob1 PUBLIC CONFIGURATIONS Release)
 
     # /RTC1 is incompatible with /await
-    if((NOT HPX_WITH_AWAIT) AND (NOT HPX_WITH_CXX20_COROUTINES))
-      hpx_add_target_compile_option(/RTC1 PUBLIC CONFIGURATIONS Debug)
-    else()
+    if(HPX_WITH_CXX20_COROUTINES)
       hpx_remove_target_compile_option(/RTC1 PUBLIC CONFIGURATIONS Debug)
+    else()
+      hpx_add_target_compile_option(/RTC1 PUBLIC CONFIGURATIONS Debug)
     endif()
```

Comment:
I will verify if this is still needed for the C++20 coroutines. I think this was a workaround for a limitation of the pre-C++20 implementation in MSVC.

Reply:
Thanks!

Reply:
Thanks for checking! I can remove them. However, should the `/RTC1` flag still be added, just not conditionally? Because it is added if coroutines aren't available. Or is there no need for the flag at all anymore?

Reply:
The flag is automatically added by CMake, we don't have to do anything. Originally we have had it in here to make sure the flag was re-added whenever the await support was disabled (without deleting the CMake cache).

Reply:
Ah, gotcha. I've removed it all.